packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CancelOrder/template.tsx

```diff
@@ -52,9 +54,10 @@ const Template: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
           >
             <FormattedMessage
               id='modals.simplebuy.cancelorder.outcome'
-              defaultMessage='Cancelling this {pair} Buy will remove your order. You can always create a new order from the menu if you cancel now.'
+              defaultMessage='Cancelling this {pair} {orderType} will remove your order. You can always create a new order from the menu if you cancel now.'
               values={{
-                pair: props.order.pair
+                pair: props.order.pair,
+                orderType: orderType === 'BUY' ? 'Buy' : 'Sell'
```

Comment:
Does this work on, say, spanish?

Reply:
this is valid point, we have such hardcoded text on several pages, I'll change this to be fully dynamic and we will see for other points

Reply:
I don't know. You tell me! If this is common patter in the repo, let's go with it!

Reply:
Heh it's follow up of this part https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/template.success.tsx#L325

Reply:
@dsisto-bc let's go like this for now, tnx!

Reply:
good call out @dsisto-bc. we have bad patterns like this all over. need to address in the future